src/style.css

```diff
@@ -0,0 +1,100 @@
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+:root {
+  --main-text-color: #000;
+  --main-border-color: #e1e7ed;
+}
+
+.bar {
+  display: block;
+  box-sizing: border-box;
+  position: relative;
+  width: 100%;
+  border: 1px solid var(--main-border-color);
+  border-radius: 4px;
+  background: #fff;
+  cursor: pointer;
+  margin-top: 20px;
+  /* box-shadow: 0px 1px 8px rgba(61, 78, 97, 0.1); */
+}
+
+.bar-big {
+  height: 70px;
+}
+
+.bar-small {
+  height: 42px;
+}
+
+.icon {
+  float: left;
+  position: absolute;
+}
+
+.icon-big {
+  width: 19px;
+  height: 19px;
+  left: 25px;
+  top: 24px;
+}
+
+.icon-small {
+  width: 11px;
+  height: 11px;
+  left: 12px;
+  top: 14px;
+}
+
+.field {
+  font-family: inherit;
+  border: none;
+  position: absolute;
+  width: 85%;
+  cursor: inherit;
+}
+
+.field-big {
+  font-size: 16px;
+  font-weight: 300;
+  height: 22px;
+  line-height: 22px;
+  top: 23px;
+  left: 61px;
+}
+
+.field-small {
+  font-size: 14px;
+  font-weight: 400;
+  height: 19px;
+  line-height: 19px;
+  top: 10px;
+  left: 32px;
+}
+
+.field::placeholder {
+  color: #3d4e61;
+}
+
+.bar:hover {
+  color: var(--main-text-color);
+  /* box-shadow: 0px 3px 8px rgba(61, 78, 97, 0.2); */
```

Comment:
The box-shadow property is commented out. If you want to apply the box-shadow on hover, uncomment this line.

Reply:
The same past answer